test/integration/integration-test.ts

```diff
@@ -74,15 +75,15 @@ function testTransaction(
   const signedData = testcase.client.sign(txJSON, secret)
   console.log('PREPARED...')
   return testcase.client
-    .submit(signedData.signedTransaction)
+    .request({command: "submit", tx_blob: signedData.signedTransaction})
```

Comment:
Out of the scope of the PR (handled in #1550)

Reply:
Merged to include #1550's changes